src/python/pants/backend/python/goals/pytest_runner.py

```diff
@@ -323,7 +323,8 @@ async def setup_pytest_for_target(
 
 @rule(desc="Run Pytest", level=LogLevel.DEBUG)
 async def run_python_test(
-    field_set: PythonTestFieldSet, test_subsystem: TestSubsystem, pytest: PyTest
+    field_set: PythonTestFieldSet,
```

Comment:
Unrelated, but I happened to notice that this rule was unnecessarily requesting PyTest, and it didn't seem worth its own PR.